homework_weekend7/src/task1/LinkedListBoolean.java

```diff
@@ -0,0 +1,198 @@
+package task1;
+
+import java.util.NoSuchElementException;
+
+/**
+ * Represents implementation of LinkedList for boolean values. Implements common
+ * methods related to the LinkedList.
+ * 
+ * @author ajla
+ *
+ */
+
+public class LinkedListBoolean {
+	private Node head;
+
+	/**
+	 * Adds element at the end of the list.
+	 * 
+	 * @param value
+	 * @return
+	 */
+	public void add(boolean value) {
```

Comment:
Method signature is not valid, as per task definition it should be `public void add(Boolean value) {}`